crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -341,6 +341,10 @@ impl CumulocityConverter {
                 let ancestors_external_ids =
                     self.entity_store.ancestors_external_ids(entity_topic_id)?;
 
+                if entity_topic_id.is_bridge_health_topic() {
```

Comment:
I'm very confused as to why I needed to make this change twice to actually get the bridge to not appear in c8y. Both `converter.rs` and `service_monitor.rs` appear to publish status messages for services, with `converter.rs` doing it relatively cleanly, while `service_monitor.rs` is publishing both `down` & `up` messages when a service starts.